packages/fast-components-react-base/src/checkbox/checkbox.tsx

```diff
@@ -87,6 +87,62 @@ class Checkbox extends Foundation<ICheckboxHandledProps & ICheckboxManagedClasse
 
         return super.generateClassNames(classes);
     }
+
+    /**
+     * Generates the correct attributes
+     */
+    private generateAttributes(): object {
+        const attributes: object = {};
+
+        /* tslint:disable:no-string-literal */
+        if (this.props.disabled) {
```

Comment:
you should just put this in the render method: `disabled={this.props.disabled || null}`